app/controllers/sessions_controller.rb

```diff
@@ -24,16 +24,13 @@ def edit
   # POST /sessions
```

Comment:
remove these changes - you can do 
`git checkout master app/controllers/sessions_controller.rb`